src/frameworks/index.ts

```diff
@@ -513,6 +514,12 @@ ${firebaseDefaults ? `__FIREBASE_DEFAULTS__=${JSON.stringify(firebaseDefaults)}\
         // continue
       });
 
+      if (fileExistsSync(getProjectPath(".npmrc"))) {
+        await copyFile(getProjectPath(".npmrc"), join(functionsDist, ".npmrc")).catch(() => {
```

Comment:
I think we can drop the catch and let the error throw

Reply:
Otherwise looks good. Mind adding a changelog entry?

Reply:
No worries, done!